CRM/Utils/System/Drupal8.php

```diff
@@ -908,4 +908,11 @@ public function suppressProfileFormErrors():bool {
     return FALSE;
   }
 
+  /**
+   * @inheritdoc
+   */
+  public function viewsIntegration(): string {
+    return '<p><strong>' . ts('To enable CiviCRM Views integration, install the <a href="https://www.drupal.org/project/civicrm_entity">CiviCRM Entity</a> module.') . '</strong></p>';
```

Comment:
Great, thanks. [Docs](https://docs.civicrm.org/dev/en/latest/translation/#best-practices) say this should be `ts('To enable CiviCRM Views integration, install the <a %1>CiviCRM Entity</a> module.', [1 => 'href="https://www.drupal.org/project/civicrm_entity"'])` but can update later.

Reply:
I'll change that now since it's a small thing.